src/style.css

```diff
@@ -0,0 +1,41 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  height: 16px;
+}
+
+.stars__star {
+  margin-right: 4px;
+  padding: 0;
+  width: 16px;
+  height: 16px;
+  background-image: url("./images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 > .stars__star:nth-child(-n + 1) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--2 > .stars__star:nth-child(-n + 2) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--3 > .stars__star:nth-child(-n + 3) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--4 > .stars__star:nth-child(-n + 4) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--5 > .stars__star:nth-child(n) {
+  background-image: url("./images/star-active.svg");
+}
```

Comment:
Don't repeat yourself, combine selectors for same styles.

Reply:
selector combined - resolved